src/test/test-data/array_tuple_multi.json

```diff
@@ -0,0 +1,10 @@
+{ "x" : [ { "1" : [ { "1" : 111, "2": [ 1, 2 ] },
```

Comment:
Should all of these test files be under the `unit/io` folder? I noticed there is a `test_csv_files` directory there, so it seems like a natural place

Reply:
moved to directory `unit/io/test_json_files`